doc/man3/EVP_PKEY_type.md

```diff
@@ -0,0 +1,43 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_type,
+EVP_PKEY_id,
+EVP_PKEY_base_id,
+EVP_PKEY_set_type,
+EVP_PKEY_set_type_str,
+EVP_PKEY_set_alias_type,
```

Comment:
I think we should either document all of these, or not include them in the pod at all (and remove them from the synopsis below). If you go with the latter of those two options then rename the file to match the one function that you are documenting. The problem is if we list these undocumented functions here then our scripts for finding undocumented functions will stop reporting them!

Reply:
OK

Reply:
Turns out EVP_PKEY_type, EVP_PKEY_id and EVP_PKEY_base_id were already documented, just not where I expected (EVP_PKEY_set1_RSA). I will add the documentation of EVP_PKEY_set_alias_type to this same doc so at least they are all together.